src/sentry/static/sentry/app/views/organizationDiscover/styles.jsx

```diff
@@ -38,6 +38,16 @@ export const PageTitle = styled.h2`
   height: ${HEADER_HEIGHT}px;
 `;
 
+export const ResultTitle = styled(Flex)`
+  flex: 1;
+  min-width: 70px;
+`;
+
+//TODO: Add @media for responsiveness
+export const ResultViewButtons = styled('div')`
+  flex: flex-end;
```

Comment:
Is this a valid property for `flex`?

Reply:
it worked, but I don't think it is. Changing it to `justify-content`, thanks!

Reply:
Another question is, is it even needed?

Reply:
Oh, thanks! I did at one point, not anymore.